### ISSUE TITLE
Low resolution on Mac version

### ISSUE THREAD

**Issue by edianannink** [marcelstoer/nodemcu-pyflasher#52]
The application has a very low resolution on my MacBook running Mojave 10.14.6.
<img width="1552" alt="Screen Shot 2019-07-26 at 11 48 40" src="https://user-images.githubusercontent.com/28990294/61943360-89e83900-af9b-11e9-92da-239a09fad8c3.png">

**Comment by marcelstoer** [comment]
Yep, I know. It has something to do with how PyInstaller mangles wxPython into the executable (I guess).

**Comment by marcelstoer** [comment]
v5.1 should fix this, please verify, thanks.

https://github.com/marcelstoer/nodemcu-pyflasher/releases/tag/v5.1.0
